joern-cli/frontends/gosrc2cpg/src/test/scala/io/joern/go2cpg/passes/ast/TypeDeclMethodCallTests.scala

```diff
@@ -45,5 +45,14 @@ class TypeDeclMethodCallTests extends GoCodeToCpgSuite {
       x.typeFullName shouldBe "string"
       x.dispatchType shouldBe DispatchTypes.STATIC_DISPATCH
     }
+
+    "check this/receiver argument is correctly getting passed" in {
+      cpg.call("fullName").argument.size shouldBe 1
+      val List(x) = cpg.call("fullName").argument.l
```

Comment:
You can save yourself the effort of checking types later down the line with
```suggestion
      val List(x: Identifier) = cpg.call("fullName").argument.l: @unchecked
```

Reply:
done